usaspending_api/etl/es_transaction_template.json

```diff
@@ -117,22 +123,17 @@
           "index": false
         },
         "award_amount": {
-          "type": "scaled_float",
-          "scaling_factor": 100
+          "type": "float"
```

Comment:
These should remain `scaled_float`

Reply:
Got reindexing errors for this field only. It works and does not effect performance. We've been at odds about this for weeks but I have yet to see what you are talking about. @MelvinDunn

Reply:
Since we were currently using `scaled_float`, the burden of proof lies with the developer to show and explain why the change is necessary if asked.

Reply:
I presented cases where the results were similar and had 0 effect on API calls and results. Would you like to see errors when I try to reindex it?